fjord/feedback/static/css/generic_feedback.less

```diff
@@ -135,19 +135,19 @@ body.html-rtl header .header-content .left-side {
 
 #back-button-container {
   display: none;
-  height: 30px;
-  margin: 0;
+  height: 38px;
+  margin: 0px 0px 0px 0px;
   padding: 0;
-  width: 30px;
+  width: 50px;
   z-index: 1;
 
   #back-button {
     cursor: pointer;
-    height: 30px;
-    margin: 0;
+    height: 38px;
+    margin: 0px;
     padding: 0;
```

Comment:
Above, you changed the left margin of `left-side` from `5px` to `0`. Here, you should change the padding of the `back-button` to `padding: 0 5px;`. That increases the touch target of the back button 5px left and 5px right without moving or resizing the image for the back button at all.